samples/otlptraceexport/package.json

```diff
@@ -24,7 +24,7 @@
     "@opentelemetry/exporter-trace-otlp-grpc": "0.52.1",
     "@opentelemetry/exporter-trace-otlp-proto": "0.52.1",
     "@opentelemetry/exporter-trace-otlp-http": "0.52.1",
-    "@grpc/grpc-js": "1.10.11",
```

Comment:
This was causing typescript errors around protected members in credentials class because npm was installing was using a different version from carat dependencies in the rest of the workspace. Not sure why renovate hasn't been sending PRs to update this version